src/styles/cyverseTheme.js

```diff
@@ -1,113 +1,22 @@
-import * as colors from 'material-ui/styles/colors';
```

Comment:
Work on this file keeps overrides to MUI components here but moves CY-UI style definitions to the styles file and brings them in at once.